File: workspace_movil/app/src/main/java/com/flotix/model/Cliente.kt
package com.flotix.model

import java.io.Serializable

class Cliente(
    var nif: String = "",
    var nombre: String = "",
    var direccion: String = "",
    var poblacion: String = "",
    var personaContacto: String = "",
    var tlfContacto: String = "",
    var email: String = "",
    var idMetodoPago: String = "",
    var cuentaBancaria: String = "",
    var baja: Boolean = false
) : Serializable {
}

File: workspace_presentacion/Flotix2021/Flotix2021/Model/Cliente.cs

namespace Flotix2021.Model
{
    /// <summary>
    /// Objeto Cliente para mandar al servidor
    /// </summary>
    class Cliente
    {
        public string nif { get; set; }
        public string nombre { get; set; }
        public string direccion { get; set; }
        public string poblacion { get; set; }
        public string personaContacto { get; set; }
        public string tlfContacto { get; set; }
        public string email { get; set; }
        public string idMetodoPago { get; set; }
        public string cuentaBancaria { get; set; }
        public bool baja { get; set; }
	}
}
